src/Selenium2Library/keywords/_screenshot.py

```diff
@@ -17,49 +17,96 @@ def __init__(self):
     def set_screenshot_directory(self, path, persist=False):
         """Sets the root output directory for captured screenshots.
 
-        ``path`` argument specifies the absolute path where the screenshots should
-        be written to. If the specified ``path`` does not exist, it will be created.
-        Setting ``persist`` specifies that the given ``path`` should
-        be used for the rest of the test execution, otherwise the path will be restored
-        at the end of the currently executing scope.
+        ``path`` argument specifies the absolute path where the screenshots
+        should be written to. If the specified ``path`` does not exist,
+        it will be created. Setting ``persist`` specifies that the given
+        ``path`` should be used for the rest of the test execution, otherwise
+        the path will be restored at the end of the currently executing scope.
         """
         path = os.path.abspath(path)
         self._create_directory(path)
         if persist is False:
             self._screenshot_path_stack.append(self.screenshot_root_directory)
             # Restore after current scope ends
-            utils.events.on('scope_end', 'current', self._restore_screenshot_directory)
+            utils.events.on('scope_end', 'current',
+                            self._restore_screenshot_directory)
```

Comment:
Are these changes here to avoid lines longer than 80 chars? That is OK in general, but such changes should be submitted separately. Preferably so that the whole module is gone through in one go. In minor cases it's also a bit questionable is the slightly enhanced PEP-8 compatibility worth an extra change in the version history.